src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/payment/BamboraPaymentAdapter.java

```diff
@@ -38,6 +39,10 @@ public EfilingTransaction makePayment(EfilingPayment efilingPayment) {
         try {
             PaymentResponse response = paymentsApi.makePayment(paymentRequest);
             result.setEcommerceTransactionId(BigDecimal.valueOf(response.getMessageId()));
+            result.setEntDtm(DateUtils.getCurrentXmlDate());
+            result.setTransactonDtm(DateUtils.getCurrentXmlDate());
```

Comment:
I think we should avoid manipulating any xml date in the api, could we move that down to the library ?